### PR TITLE
Provide an endpoint aggregating the average stats on the fetcher.

### PR DESCRIPTION
**Pull request by FunkySayu**

### CODE REVIEW

File: powerspikegg/rawdata/fetcher/aggregator_test.py [hunk 1]
```diff
@@ -122,6 +167,183 @@ def test_champion_and_league_filtering(self):
...
+            "wardsKilled": 2,
+            "wardsPlaced": 7,
+            "totalScoreRank": 0,
+            "totalDamageTaken": 10852,
+        }
+        result = aggregator.AverageStatisticsOnQuery(self.collection, query)
+        self.assertDictEqualWithDebug(result, expected)
+
+    def test_avg_league_and_champions_filtering(self):
+        """Tests filter per champion on the average aggregation."""
```
Comment (ArchangelX360) [inline powerspikegg/rawdata/fetcher/aggregator_test.py]: Should be "per champion AND league" in comment.

Comment (ArchangelX360) [reply powerspikegg/rawdata/fetcher/aggregator_test.py]: Also, every "... played are selectionned." line needs to be re-written (I can't comment on these as it is not on this PR).

**Review by ArchangelX360 — CHANGES_REQUESTED** [review]
"selectionned" ;) #revenge

**Comment by FunkySayu** [comment]
Fixed.

**Review by ArchangelX360 — APPROVED** [review]
LGTM.

**Pull request merged**